### DISCUSSION TITLE
Rotating Data in IPF-Maps

### DISCUSSION THREAD

**Discussion by mardilos** [mtex-toolbox/mtex#1798] [Ask Anything]
Dear all,

i want to rotate my data in the ipf-maps according to the rotation which was done by centering the sample in the ODF (MTEX 5.3)

as you can see in the attached image, the centering of the PF works in Z directions. (Seems that in x direction a small misorientation is still theresince not perfectly 110 aligned)

When i then apply the rotation to my grain data, the orientation changes as excpected due to the color code in the IPF-map, but it seems that MTEX rotated 'the whole sample' and not just the orientation data.

I am not sure what i am missing here...

Best
m

[Rotation_code.zip](
![Rotating data](https://user-images.githubusercontent.com/65020763/233409671-3cd8df3a-fba9-42bd-b3b9-d5612465a4e9.png)
https://github.com/mtex-toolbox/mtex/files/11287757/Rotation_code.zip)

**Comment by lfgmorales** [comment]
See here for potential flags to be used when doing the rotations

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 20 Apr 2023, at 17:12, mardilos ***@***.***> wrote:

﻿

Dear all,

i want to rotate my data in the ipf-maps according to the rotation which was done by centering the sample in the ODF (MTEX 5.3)

as you can see in the attached image, the centering of the PF works.

When i then apply the rotation to my grain data, the orientation changes as excpected due to the color code in the IPF-map, but it seems that MTEX rotated 'the whole sample' and not just the orientation data.

I am not sure what i am missing here...

Best
m

[Rotation_code.zip](
[Rotating data]<https://user-images.githubusercontent.com/65020763/233409671-3cd8df3a-fba9-42bd-b3b9-d5612465a4e9.png>
https://github.com/mtex-toolbox/mtex/files/11287757/Rotation_code.zip)

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1798>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3OYTKSH36J7JAEKIDXCFG6PANCNFSM6AAAAAAXFUXNUE>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1798", "url": "https://github.com/mtex-toolbox/mtex/discussions/1798", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by mardilos** [reply]
Hi Luiz,
with flags you mean i missed an additinal comment when i do rotation?

i just calulated the grains again from the rotated ebsd data, not sure if this is the correct way

% Calculate a list of grains
[grains_corr, ebsd_corr.grainId] = calcGrains(ebsd_corr,'angle',10*degree)
